app/views/idv/in_person/address.html.erb

```diff
@@ -1,15 +1,92 @@
-  <%= t('in_person_proofing.headings.address') %>
-</h1>
-
-<%= validated_form_for :doc_auth,
-                       url: url_for,
-                       method: 'put',
-                       html: { autocomplete: 'off', class: 'margin-y-5' } do |f| %>
-  <%= f.button :button,
-               t('doc_auth.buttons.continue'),
-               type: :submit,
-               class: 'usa-button--big usa-button--wide' %>
-<% end %>
+<%= render PageHeadingComponent.new.with_content(t('in_person_proofing.headings.address')) %>
+
+<p>
+  <%= t('in_person_proofing.body.address.info_html') %>
+  <%= new_window_link_to(t('in_person_proofing.body.address.learn_more'), MarketingSite.security_and_privacy_practices_url) %>
+</p>
+
+<div class="margin-top-4 margin-bottom-4">
+  <%= validated_form_for(
+        :doc_auth, url: url_for, method: 'PUT',
+                   html: { autocomplete: 'off', class: 'margin-top-2' }
+      ) do |f| %>
+    <div class="margin-bottom-3">
+      <%= render ValidatedFieldComponent.new(
+            name: :address1,
+            form: f,
+            label: t('in_person_proofing.form.address.address1'),
+            required: true,
+            maxlength: 255,
+            input_html: { aria: { invalid: false }, value: pii['address1'] },
+          ) %>
+    </div>
+    <div class="margin-bottom-3">
+      <%= render ValidatedFieldComponent.new(
+            name: :address2,
+            form: f,
+            label: t('in_person_proofing.form.address.address2'),
+            required: false,
+            maxlength: 255,
+            input_html: { value: pii['address2'] },
+          ) %>
+    </div>
+    <div class="margin-bottom-3">
+      <%= render ValidatedFieldComponent.new(
+            name: :city,
+            form: f,
+            label: t('in_person_proofing.form.address.city'),
+            required: true,
+            maxlength: 255,
+            input_html: { aria: { invalid: false }, class: 'usa-input', value: pii['city'] },
+          ) %>
+    </div>
+
+    <%= render ValidatedFieldComponent.new(
+          name: :state,
+          form: f,
+          collection: us_states_territories,
+          label: t('in_person_proofing.form.address.state'),
+          prompt: t('in_person_proofing.form.address.state_prompt'),
+          required: true,
+          selected: pii['state'],
+        ) %>
+
+    <div class="tablet:grid-col-6 margin-bottom-3">
+      <%# using :tel for mobile numeric keypad %>
+      <%= render ValidatedFieldComponent.new(
+            name: :zipcode,
+            form: f,
+            as: :tel,
+            label: t('in_person_proofing.form.address.zipcode'),
+            required: true,
+            pattern: '(\d{5}([\-]\d{4})?)',
+            input_html: { aria: { invalid: false }, class: 'usa-input', value: pii['zipcode'] },
+            error_messages: { patternMismatch: t('idv.errors.pattern_mismatch.zipcode') },
+          ) %>
+    </div>
+
+
+    <div class="margin-top-0">
```

Comment:
A `div` has no default margin, is the `margin-top-0` necessary?

```suggestion
    <div>
```

(If the `<div>` can be removed altogether, all the better)